python/ray/util/sgd/torch/training_operator.py

```diff
@@ -154,6 +197,13 @@ def train_epoch(self, ...):
 
         return metric_meters.summary()
 
+    def forward(self, features, target):
+        output = self.model(features)
+        return self.criterion(output, target)
```

Comment:
Can we remove this? this isn't fundamental right?

Reply:
Probably should remove from this PR, but overall I think it's a good idea to add it at some point. I'll motivate in another PR